docs/v1/api.md

```diff
@@ -0,0 +1,230 @@
+# Service Catalog API
+
+This document proposes a Kubernetes-native service broker API. It is a formalization of [@krancour](http://github.com/krancour)'s [presentation](https://docs.google.com/presentation/d/1Fm3qG9zQ4R8ZbhtTObmQjO3UZYJ0ruYI4qr5w_gEKN8/edit?usp=sharing) that he gave at the sig-service-catalog meeting on 10/10/2016.
+
+# Motivations
+
+2 different demos were given at [10/3/2016’s sig-service-catalog meeting](https://www.youtube.com/watch?v=Kfar5Uw7CRg), both extremely promising. Each differed in quite a few ways, including how consumers used Kubernetes resources to communicate.
+
+This proposal aims to define a standard API, including a schema for Kubernetes-native resources, to:
+
+- Guide the continuing efforts shown in those demos
+- Improve the user experience (both cluster operator and application developer)
+- Attempt to extend the Kubernetes API instead of relying on comparatively clunky Third Party Resource types (long-term goal)
+- Ensure a consistent user experience regardless of implementation, without undermining competition or choice
+
+# Inspiration
```

Comment:
I have no problem with that. Changed in dfb571e8ff1db16891a05c407edf28c87dcdca23